test/services/api/handlers/projects.js

```diff
@@ -1135,29 +1135,6 @@ experiment('Project Handlers', function() {
       });
     });
 
-    test('update thumbnail (320) succeeds', function(done) {
```

Comment:
these are all invalid, since we're not setting thumbnails on this route anymore.